sprucelabsai/sprucebot-cli
# [SB-773] User is reminded to set ngrok tunnel to https. Input is not …

…accepted if it is not.

[SB-773](https://jira.sprucelabs.ai/jira/browse/SB-773)

@sprucelabsai/engineers

## Description 
User is now reminded to set ngrok tunnel to https.
Input is not accepted if the tunnel is not https and will continue to remind the user until it is https.

## Type

## Steps to Test or Reproduce
Register a new skill.  When it asks for you to set your tunnel, notice it reminds you again to set it to https.  
Then try to put in the http of your tunnel.
It will not accept it, remind you to use https, and give you a chance to set your tunnel again.
This time, use the https.  The skill then registers.
